src/doc/book/closures.md

```diff
@@ -492,12 +492,12 @@ fn factory() -> Box<Fn(i32) -> i32> {
 
     Box::new(move |x| x + num)
 }
-# fn main() {
+fn main() {
 let f = factory();
 
 let answer = f(1);
 assert_eq!(6, answer);
-# }
+}
```

Comment:
Sorry to comment when this was already approved, but it'd read nicer better the main function's body was indented and there was an empty line in front of the fn.